sqlparser/parser.go

```diff
@@ -473,18 +473,41 @@ tailloop:
 		case tt == ReservedWordToken && s.Token() == "as":
 			CopyToken(s, &result.Body)
 			NextTokenCopyingWhitespace(s, &result.Body)
-			if firstAs {
-				// Add the `RoutineName` token as a convenience, so that we can refer to the procedure/function name
-				// from inside the procedure (for example, when logging)
-				if result.CreateType == "procedure" {
-					procNameToken := Unparsed{
-						Type:     OtherToken,
-						RawValue: fmt.Sprintf(templateRoutineName, strings.Trim(result.QuotedName.Value, "[]")),
+			/*
```

Comment:
Leaving as a comment because the idea of having `RoutineName` is good.  But we need to figure out what is going on with `rowcount` first.